charts/incubator/cloudflared/ci/test-values.yaml

```diff
@@ -1,3 +1,9 @@
-# With the bellow we test both container starts and that tpl is working inside extraArgs list
-args: ["tunnel", "--no-autoupdate", "run", "{{ .Values.testTpl }}"]
-testTpl: "--hello-world"
+workload:
+  main:
```

Comment:
Duh… complete missed it on both files lol!
Nice job.

Reply:
As you can see from the commit history.... this took me forever to figure out 🤦

Reply:
Hehe. I’ve been fighting over a ton of different formats and it gets out of sight sometime lol